### DISCUSSION TITLE
Error Configuring AzOps using Azure CLI in PowerShell

### DISCUSSION THREAD

**Discussion by petropmi** [Azure/AzOps#750] [General]
Hi everyone,

I'm facing an error each time I'm executing the PowerShell script that configures the Azure DevOps project. Although the project, repo, and the pipelines are created successfully, later on, fails at the step of assigning permissions to the Build Service account.

The error is the following:

ERROR: The command failed with an unexpected error. Here is the traceback:
ERROR: No connection adapters were found for '"https://vssps.dev.azure.com/organization/_apis/identities?searchFilter=General&queryMembership=None&api-version=6.0&filtervalue=azops-artofcloud Build Service (organization)"'.

Any ideas are welcomed!

**Comment by daltondhcp** [comment]
Hi @petropmi,

I can reproduce the problem, so something must have changed on the ADO side. 
Thus, I have created an issue #753 to track this. Meanwhile we are working on a fix, I'd recommend reverting to manual configuration.

Thank you for your understanding. 

/Johan

**Reply by petropmi** [reply]
Thank you @daltondhcp !!
